src/voterData/voterOutreach.service.ts

```diff
@@ -0,0 +1,141 @@
+import { Injectable, Logger } from '@nestjs/common'
+import { SlackService } from 'src/shared/services/slack.service'
+import { ScheduleOutreachCampaignSchema } from './voterFile/schemas/ScheduleOutreachCampaign.schema'
+import { Campaign, User } from '@prisma/client'
+import { buildSlackBlocks } from './util/voterOutreach.util'
+import { FileUpload } from 'src/files/files.types'
+import { CampaignsService } from 'src/campaigns/services/campaigns.service'
+import * as sanitizeHtml from 'sanitize-html'
+import { IS_PROD, WEBAPP_ROOT } from 'src/shared/util/appEnvironment.util'
+import { VOTER_FILE_ROUTE } from './voterFile/voterFile.controller'
+import { FilesService } from 'src/files/files.service'
+import { SlackChannel } from 'src/shared/services/slackService.types'
+
+@Injectable()
+export class VoterOutreachService {
+  private readonly logger = new Logger(VoterOutreachService.name)
+
+  constructor(
+    private readonly slack: SlackService,
+    private readonly filesService: FilesService,
+    private readonly campaignsService: CampaignsService,
+  ) {}
+
+  async scheduleOutreachCampaign(
+    user: User,
+    campaign: Campaign,
+    {
+      budget,
+      audience,
+      script,
+      date,
+      message,
+      voicemail,
+      type,
+    }: ScheduleOutreachCampaignSchema,
+    imageUpload: FileUpload,
+  ) {
+    const { firstName, lastName, email, phone } = user
+
+    // TODO: reimplement
+    // const crmCompany = await sails.helpers.crm.getCompany(campaign)
+    // if (!crmCompany) {
+    //   throw new Error(`crmCompany not found for ${campaign}`)
+    // }
+    // const assignedPa = await getCrmCompanyOwnerName(crmCompany, true)
+
+    const messagingScript: string = campaign.aiContent?.[script]?.content
+      ? sanitizeHtml(campaign.aiContent?.[script]?.content, {
+          allowedTags: [],
+          allowedAttributes: {},
+        })
```

Comment:
Shouldn't this be getting sanitized on the way _into_ the DB, not on the way out? 🤔